doc/docs.md

```diff
@@ -1030,6 +1030,42 @@ println(a) // `[2, 2, 2, 13, 2, 3, 4]`
 println(b) // `[2, 3, 13]`
 ```
 
+### Slices with negative indexes
+
+V support for array and strings slices with negative indexes.
+Negative indexing starts from the end of the string
+towards the start, for example `-3` is equal to `array.len - 3`.
+Negative slices have a different syntax from the normal ones, you need
+to add a `gate` between the array name and the square bracket: `a#[..-3]`.
+The `gate` specifies that this is a different type of slice and remember that
+the result is "locked" inside the array.
+The retuned slice is always a valid array, though it may be empty:
+```v
+a := [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+println(a#[-3..]) // [7, 8, 9]
+println(a#[-20..]) // [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+println(a#[-20..-8]) // [0, 1]
+println(a#[..-3]) // [0, 1, 2, 3, 4, 5, 6]
+
+// empty arrays
+println(a#[-20..-10]) // []
+println(a#[20..10]) // []
+println(a#[20..30]) // []
+```
+
+### Array methods concatenation
```

Comment:
Method chaining

Reply:
I am not sure, if `concatenation` is clear enough - I understand it as mechanically appending something to something else, @JalonSolov what do you think?

Reply:
I would change it to be more explicit to what it is/what it's doing:

`### Array method chaining`

Reply:
Ok